src/composables/sdk13.ts

```diff
@@ -164,6 +153,31 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     };
   }
 
+  watch(
+    activeAccount,
+    (oldVal, newVal) => {
+      if (!isEqual(oldVal, newVal) && sdk) {
+        sdk._pushAccountsToApps();
+      }
+    },
+    { deep: true },
+  );
+
+  watch(
+    activeNetwork,
+    async (oldVal, newVal) => {
+      if (!isEqual(oldVal, newVal) && sdk) {
+        sdkBlocked = true;
+        sdk.pool.delete(sdkCurrentNetwork.name);
+        const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+        sdk.addNode(activeNetwork.value.name, nodeInstance!, true);
+        sdkCurrentNetwork = activeNetwork.value;
+        sdkBlocked = false;
+      }
+    },
+    { deep: true },
+  );
```

Comment:
I consider using this code as a step back from what we have and than we refactored.
1. As this is a composable we can't be sure if it's loaded at the moment of changing the network. So this watcher may not run.
2. We should not watch the network - we need to reload the SDK only if the app needs to use the SDK and it detects that the network has changed.

Reply:
As we discussed my suggestion is to bring back the `resetSdkNode` method and temporarily use it within the scope of the store when running the  `switchNetwork` action. After migration from the Vuex we would run it within the scope of the networks composable.

Reply:
`resetSdkNode` would create on-demand SDK instances and thus will not notify the connected aepps. Plus when you re-instantiate the SDK then we would lose all connected aepps.